docs/src/pages/guides/BringYourOwnPersistence.md

```diff
@@ -78,58 +78,129 @@ If you are using Maven, add the following dependencies to your persistence layer
 
-Although the HL7 FHIR specification [doesn't strictly require all servers to support versioning](https://hl7.org/fhir/R4B/http.html#versions), the IBM FHIR Server is built to be version-aware. This means that all FHIRPersistence implementations should implement the `vread` and `history` interactions.
-Similarly, the IBM FHIR Server was written for read/write datastores and so `create` and `update` should be supported as well.
+You do not need to provide an implementation of the getPayloadPersistence call unless you intend to also implement payload offloading (used in cases where the resource payload object is stored outside the primary database).
+
+Although the HL7 FHIR specification [doesn't strictly require all servers to support versioning](https://hl7.org/fhir/R4B/http.html#versions), the LinuxForHealth FHIR Server is built to be version-aware. This means that all FHIRPersistence implementations should implement the `vread` and `history` interactions.
+Similarly, the LinuxForHealth FHIR Server was written for read/write datastores and so `create` and `update` should be supported as well.
 If you have a use case for a read-only or non-version-aware server, please contact us and consider contributing the necessary modifications to the server to make this supported.
 
-The IBM FHIR Server does support persistence implementations which do not support `delete` or transactions (e.g. for transaction bundles), so please implement `FHIRPersistence.isTransactional()` and `isDeleteSupported()` accordingly.
+The LinuxForHealth FHIR Server does support persistence implementations which do not support `delete` or transactions (e.g. for transaction bundles), so please implement `FHIRPersistence.isTransactional()` and `isDeleteSupported()` accordingly.
 
 #### Create
-Create requests include a FHIRPersistenceContext and an already-validated instance of the resource to create. FHIRPersistence implementations are responsible for adding the `Resource.id`, `Resource.meta.lastUpdate`, and `Resource.meta.versionId` elements to the resource before storing it. Implementations must return a SingleResourceResult with the result of the interaction.
+```
+    /**
+     * Stores a new FHIR Resource in the datastore. The resource is not modified before it is stored. It
+     * must therefore already include correct Meta fields.
+     *
+     * @param context the FHIRPersistenceContext instance associated with the current request
+     * @param resource the FHIR Resource instance to be created in the datastore
+     * @return a SingleResourceResult with a ResourceResult that holds the unmodified resource and/or
+     *         an OperationOutcome with hints, warnings, or errors related to the interaction
+     * @throws FHIRPersistenceException
+     */
+    <T extends Resource> SingleResourceResult<T> create(FHIRPersistenceContext context, T resource) throws FHIRPersistenceException;
```

Comment:
Its nice having these here, but it will also be more likely to go stale.  Think we can remember to update these if we ever change the interface?

Reply:
Yeah, same thought crossed my mind...but then I noticed we were describing the parameters in the text anyway, so I thought it better to be clear and include the actual javadoc. We know that if we change FHIRPersistence we need to update this document, so I think it's OK to have this here.